Description: Fix missing aarch64 syscalls
Origin: Backport of https://github.com/seccomp/libseccomp/pull/235
Author: Alex Murray <alex.murray@canonical.com>
--- a/src/arch-aarch64-syscalls.c
+++ b/src/arch-aarch64-syscalls.c
@@ -68,7 +68,7 @@ const struct arch_syscall_def aarch64_sy
 	{ "clock_settime", 112 },
 	{ "clock_settime64", __PNR_clock_settime64 },
 	{ "clone", 220 },
-	{ "clone3", __PNR_clone3 },
+	{ "clone3", 435 },
 	{ "close", 57 },
 	{ "connect", 203 },
 	{ "copy_file_range", 285 },
@@ -159,7 +159,7 @@ const struct arch_syscall_def aarch64_sy
 	{ "getresgid32", __PNR_getresgid32 },
 	{ "getresuid", 148 },
 	{ "getresuid32", __PNR_getresuid32 },
-	{ "getrlimit", __PNR_getrlimit },
+	{ "getrlimit", 163 },
 	{ "getrusage", 165 },
 	{ "getsid", 156 },
 	{ "getsockname", 204 },
@@ -391,7 +391,7 @@ const struct arch_syscall_def aarch64_sy
 	{ "setresuid32", __PNR_setresuid32 },
 	{ "setreuid", 145 },
 	{ "setreuid32", __PNR_setreuid32 },
-	{ "setrlimit", __PNR_setrlimit },
+	{ "setrlimit", 164 },
 	{ "setsid", 157 },
 	{ "setsockopt", 208 },
 	{ "settimeofday", 170 },
